src/uu/env/src/env.rs

```diff
@@ -602,6 +655,44 @@ fn apply_specified_env_vars(opts: &Options<'_>) {
     }
 }
 
+fn apply_ignore_signal(opts: &Options<'_>) -> UResult<()> {
+    for sig in &opts.ignore_signal {
+        let sig_str = match sig.to_str() {
+            Some(s) => s,
+            None => {
+                return Err(USimpleError::new(
+                    1,
+                    format!("invalid signal name: {}", sig.quote()),
+                ))
+            }
+        };
+        let sig_value = parse_signal_value(sig_str)?;
+        let sig: Option<Signal> = if sig_str == "STOP" {
```

Comment:
Why does SIGSTOP need to be treated especially? Instead of trying to predict an error that might occur later, I suggest that we instead simply try to set the signal handler, and only deal with it *then*.

Reply:
Signals for which `sig action` cannot be assigned e.g. `SIGKILL`,`SIGSTOP` have been handled in `ignore_signal` function.